docs/RELEASING.md

```diff
@@ -1,221 +1,30 @@
 # Releasing D2-ui packages
 
-## TL;DR
+The root-level `package.json` contains the `version` key which will be
+used to publish all packages.
 
-```
-$ yarn install
-$ yarn release
-$ lerna publish
-```
-
-## Step 0: Before starting
-
-Make sure that you have run the `install` process to make sure that
-everything works as intended. Also double check the example application
-that everything looks OK.
-
-## Step 1: Building
-
-There is a shortcut command to run the `bootstrap`, `build`,
-`build:umd`, and `test`, commands in sequence:
-
-```
-yarn release
-```
-
-## Step 3: Publish
-
-Familiarise yourself with [Lerna](http://lernajs.io/) before proceeding.
-
-You will need it installed globally.
-
-Before running the following lerna commands, make sure you have the latest git tags locally
-```
-git fetch --tags
-```
+The release process has been simplified as to be executed by Travis
+only using [deploy-build](https://github.com/dhis2/deploy-build).
 
-Use the `updated` and `diff` commands to get an idea of what Lerna thinks should be updated.
+# Deployment
 
-```
-$ lerna updated     # used by publish to determine changes
+All commits across all branches is built automatically and published to
+[d2-ci/d2-ui-<package>](https://github.com/d2-ci?utf8=%E2%9C%93&q=d2-ui-&type=&language=)
+on a successful build using the `deploy-build.sh` script.
 
-$ lerna diff        # used internally to diff against the last released tag and
-                    # prints the code changes which is useful for determining semver
-```
+# Publish to NPM
 
-To avoid creating tags, pushing them, etc. only to fail on the actual
-publishing to NPM make sure you are logged in.
+To publish to Github the developer needs only to create a version tag
```

Comment:
To publish to NPM, create a version tag locally using ...

Reply:
Thanks, will fix.